dnf/cli/commands/__init__.py

```diff
@@ -154,7 +154,7 @@ class InfoCommand(Command):
     info command.
     """
 
-    aliases = ('info',)
+    aliases = ('info','if', )
```

Comment:
As pep8 said, space after comma. And you don't need that `, ` at the end anymore.

Reply:
Fixed.